tests/exchange/test_exchange.py

```diff
@@ -2700,12 +2795,17 @@ def test_name(default_conf, mocker, exchange_name):
     assert exchange.id == exchange_name
 
 
+@pytest.mark.parametrize("trading_mode,amount", [
+    ('spot', 0.2340606),
+    ('futures', 23.40606),
```

Comment:
On the most recent commit I changed `('futures', 2.340606),` to `('futures', 23.40606),`

I don't understand why it was passing before

Reply:
The test was correct - your rebase created another bug (see last commit) - calling the conversion twice.